src/nyc_trees/apps/users/views/group.py

```diff
@@ -206,3 +206,12 @@ def _grant_mapping_access(group, username, is_approved):
     return {
         'mapper': mapper
     }
+
+
+def request_mapper_status(request):
+    mapper, created = TrustedMapper.objects.update_or_create(
```

Comment:
Would it be better to have this view raise and return a `4xx` code if executed repeatedly?